angular.json

```diff
@@ -29,6 +30,12 @@
           },
           "configurations": {
             "production": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
```

Comment:
If I understand correctly, it shows error during the angular build process if styles exceed 6kb (https://angular.io/guide/build)? Why do we need this since we are not adding any styles at all?

Reply:
Yup, it isn't necessary for the build, so we can remove it.